homeassistant/helpers/config_validation.py

```diff
@@ -80,9 +80,10 @@ def isfile(value: Any) -> str:
     return file_in
 
 
-def ensure_list(value: Union[T, Sequence[T]]) -> List[T]:
+def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     """Wrap value in list if it is not one."""
-    return value if isinstance(value, list) else [value]
+    return (value if isinstance(value, list) else
```

Comment:
I don't like that `ensure_list` now has a default value (`[]`). I would prefer instead to keep it to the original implementation.